jekyll/_sass/_instantsearch.scss

```diff
@@ -15,25 +15,46 @@
   top: 70px;
 
   #search-box {
-    max-width: 1200px;
-    margin: 0 auto;
+    position: relative;
     display: block;
+    max-width: 400px;
+    margin-left: 18px;
+    margin-right: 18px;
 
-    .ais-search-box {
-      max-width: 400px;
+    @media screen and (min-width: $screen-md) {
+      margin: 0;
+    }
+
+    .ais-SearchBox-submit,
+    .ais-SearchBox-reset {
+      position: absolute;
+      top: calc(50% - 20px/2);
 
-      @media screen and (min-width: $screen-sm) {
-        left: 6px;
+      svg {
+        width: 20px;
```

Comment:
The reset icon is 14px x 14px in the design, and also a light grey colour.
![Screen Shot 2020-10-09 at 9 39 21 AM](https://user-images.githubusercontent.com/5113432/95600580-5f5b9200-0a20-11eb-9b83-1162e6c0e49d.png)
Design:
![Screen Shot 2020-10-09 at 11 13 35 AM](https://user-images.githubusercontent.com/5113432/95600678-81551480-0a20-11eb-854c-6ff6ef0d7678.png)